main.go

```diff
@@ -739,3 +756,20 @@ func validateHostname(hostname string) bool {
 	}
 	return true
 }
+
+func uploadToCloudflare(hostnames string, certificate, privateKey []byte) (err error) {
+	// init cf
+	authentication := APIAuthentication{Key: *cfAPIKey, Email: *cfAPIEmail}
+	cf := NewCloudflare(authentication)
+
+	// loop hostnames
+	hostnameList := strings.Split(hostnames, ",")
+	for _, hostname := range hostnameList {
+		_, err := cf.CreateSSLConfiguration(hostname, string(certificate), string(privateKey))
```

Comment:
Does this work for hostnames with wildcard? Particularly if you have multiple hostnames within the same domain like `*.estafette.io` and *.tst.estafette.io`? Since they'll both try to upload the same certificate with both hostnames in it.

Reply:
Yes, it works. Already confirmed with one of our websites (VW nl)

One thing that is not so nice: if you have many wildcard hostnames for the same zone (e.g. *.estafette.io, *.tst.estafette.io, *.dev.estafette.io, etc.), it will be uploading the same certificate again and again.

Reply:
Also just realized our quota is for 1 uploaded edge certificate, so every following trial will result in an error.

I will check how to work this out in a better way.